spec/requests/sign_up_spec.rb

```diff
@@ -42,4 +42,32 @@
       end
     end
   end
+  path '/login' do
+    post('give jwt') do
+      tags 'Sign_up'
+      produces 'application/json'
+      parameter name: :signup, in: :body, schema: {
+        type: :object,
+        properties: {
+          email: { type: :string },
+          password: { type: :string }
+        },
+        required: %w[email password]
+      }
+      response(200, 'successful') do
```

Comment:
Add example of successfull response here, please.

Reply:
I suggest for such case to write example directly in schema:

response(200, 'successful') do
        schema type: :object,
               properties: {
                 jwt: { type: :string, example: 'sdfdsafsdfdsdf' }
               },
               required: %w[jwt]
        run_test!
      end
      
      The exaple will be generated automatically

Reply:
Updated